### PR TITLE
Fix/7 tutorial skippable fixed

### PR DESCRIPTION
**Pull request by raulgonzalezupc**
What is new?

- Tutorial Skipable UI
- Lightball given to the player when teleported
- Bug fix that the player could use the shield while on conversation

### CODE REVIEW

**Review by yalania — APPROVED** [review]
Nice job, just two minor bugs:

- If a tutorial popup is being show it will be place overlapping with the skip screen
- You don't need to make the player translate to the return position in case of skipping the tutorial.

File: Scripts/src/Script/AllyTrigger.cpp [hunk 1]
```diff
@@ -188,3 +220,42 @@ void AllyTrigger::InitPublicGameObjects()
...
+		player_1->player_controller->SetIsInsideTrigger(false);
+		player_2->player_controller->SetIsInsideTrigger(false);
+	}
+	if (input_manager->GetGameInputDown("SkipTutorial", static_cast<PlayerID>(0)))
+	{
+		ally->do_tutorial = false;
+		ally->PlayAnimation(DialogueFeeling::FINISH);
+		tutorial_helper->tutorial_request->Disable();
+		ally->tutorial_request = false;
+		SetPlayersPositions(player_1_inside, player_2_inside);
```
Comment (yalania) [inline Scripts/src/Script/AllyTrigger.cpp:255]: You can remove the SetPlayersPositions from here I think.

Comment (raulgonzalezupc) [reply Scripts/src/Script/AllyTrigger.cpp:255]: I can do it for the don't do tutorial, but if you want to do it, does a random cut with the complain dialogue, so it's better to tp him. but i change it if don't

File: Scripts/src/Script/AllyInteractionsScript.cpp [hunk 2]
```diff
@@ -318,6 +319,8 @@ void AllyInteractionsScript::TransformIntoLightBall()
 {
 	if (!ally_is_lightball)
 	{
+		App->engine_log->Log("throw light ball");
```
Comment (mariofv) [inline Scripts/src/Script/AllyInteractionsScript.cpp:322]: Delete log before merging

File: Scripts/src/Script/AllyTrigger.cpp [hunk 3]
```diff
@@ -188,3 +221,43 @@ void AllyTrigger::InitPublicGameObjects()
...
+		player_1->player_controller->SetIsInsideTrigger(false);
+		player_2->player_controller->SetIsInsideTrigger(false);
+	}
+	if (input_manager->GetGameInputDown("SkipTutorial", static_cast<PlayerID>(0)))
+	{
+		ally->do_tutorial = false;
+		ally->PlayAnimation(DialogueFeeling::FINISH);
+		tutorial_helper->tutorial_request->Disable();
+		ally->tutorial_request = false;
+		//SetPlayersPositions(player_1_inside, player_2_inside);
```
Comment (mariofv) [inline Scripts/src/Script/AllyTrigger.cpp:257]: Commented code should be deleted

**Review by mariofv — APPROVED** [review]
LGTM

**Pull request merged**